lib/parser/provenance.h

```diff
@@ -173,6 +173,9 @@ class AllSources {
       const std::string &message, bool echoSourceLine = false) const;
   const SourceFile *GetSourceFile(
       Provenance, std::size_t *offset = nullptr) const;
+  std::optional<SourcePosition> GetSourcePosition(Provenance) const;
+  std::optional<ProvenanceRange> GetFirstFileProvenance() const;
+  ProvenanceRange GetContiguousRangeAround(ProvenanceRange) const;
```

Comment:
I think that this member function prototype is for an unused function that no longer exists.